packages/data/tests/integration/query-test.js

```diff
@@ -135,6 +135,13 @@ module('Integration | @cardstack/data service query()', function (hooks) {
       });
     });
 
+    test("it can load cards without a type", async function (assert) {
+      let dataService = this.owner.lookup('service:cardstackData');
+      let pages = await run(() => dataService.query('isolated', { filter: { title: 'Homepage' } }));
```

Comment:
Make sure to assert how many results you’d expect here. If there is only one homepage card, then assert `pages.data.length` is 1, because when filtering doesn’t work you end up with a result set  that includes all the cards. And your test doesn’t actually prove filtering works in its current form as you always just grab the first item.

Reply:
Yeah, makes sense - added an assertion on result length.